redis/redis.go

```diff
@@ -154,6 +159,49 @@ func (p *RedisProvider) Init(maxlifetime int64, configs string) (err error) {
 	return p.c.Ping().Err()
 }
 
+func (p *RedisProvider) initSentinel(cfg *ini.File) (err error) {
+	opt := &redis.FailoverOptions{}
+
+	for k, v := range cfg.Section("").KeysHash() {
+		switch k {
+		case "master_name":
+			opt.MasterName = v
+		case "sentinel_Addrs":
+			opt.SentinelAddrs = strings.Split(v, "|")
```

Comment:
Any reason you choose to use `|` over `,` as delimiter here? I feel comma is more nature choice.

Reply:
Comma will be confict with others, here are examples:

single:
```
PROVIDER_CONFIG: addr=1.2.3.4:6379,password=pass
```

sentinel:
```
PROVIDER_CONFIG: ha_mode:sentinel,master_name=mymaster,sentinel_Addrs=1.2.3.4:26379|4.3.2.1:26379,password=pass
```

Reply:
Gotcha, LGTM then.